build/source/engine/qTimeDelay.f90

```diff
@@ -68,8 +64,8 @@ subroutine qOverland(&
  ! initialize error control
  err=0; message='qOverland/'
 
- ! compute instantaneous runoff (m s-1)
- averageInstantRunoff = averageSurfaceRunoff + averageAquiferBaseflow + averageSoilBaseflow
```

Comment:
The original code was correct.

Reply:
agree ... but also duplicative of the calculation in run_oneGRU.  better to keep it in one place?

Reply:
See comment above

Reply:
I agree that it is best to remove the summation here.